### ISSUE TITLE
About "Application" category score

### ISSUE THREAD

**Issue by Violettttee** [TIGER-AI-Lab/MEGA-Bench#7]
Hi~
I see you place 8 Application score on the (leaderboard)[https://huggingface.co/spaces/TIGER-Lab/MEGA-Bench]。But in the data_with_score.json,i only see the mean_score with everysingle task, is there a script to get the 8 Application score? Thanks a lot!

**Comment by woodfrog** [comment]
Yes, please use [this script](https://github.com/TIGER-AI-Lab/MEGA-Bench?tab=readme-ov-file#multi-dimensional-breakdown-analysis) to generate the per-dimension stats as in our leaderboard.

**Comment by Violettttee** [comment]
Hi~ 
I wonder is there "all_task_meta.json" file? i search the rep but couldn't find one.
